src/components/Widget/utils.ts

```diff
@@ -0,0 +1,144 @@
+import type { JSONSchema7 as JSONSchema } from 'json-schema';
+import { UiSchema as rjsfUiSchema } from '@rjsf/core';
+import { Overwrite } from '~/common';
```

Comment:
Does `~` work here? Just checking as we didn't use it in the AutoUI folder files

Reply:
it should as it is set in the config file, but since we never used it let me change it for consistency